wandb/sdk/internal/job_builder.py

```diff
@@ -467,8 +470,6 @@ def build(
                 "warn",
             )
             return None
-        metadata["dockerfile"] = dockerfile
-        metadata["build_context"] = build_context
```

Comment:
We're removing these because we conditionally add them below yes?

Reply:
correct